bmq-sdk/src/test/java/com/bloomberg/bmq/it/PayloadIT.java

```diff
@@ -37,61 +37,6 @@ public class PayloadIT {
 
     static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    // Disable this test because it uses plain sockets and read BMQ
-    // events unreliably
-    // @Test
-    public void testPayloadPlain() throws IOException, InterruptedException {
```

Comment:
`PlainConsumerIT`, `PlainProducerIT` and this `testPayloadPlain` disabled for a long time. These producer/consumer tests were good as a low-level example, but from the comments the code is not reliable.
I propose to remove it, because we will have a high-level Consumer/Producer tutorials and also convenient testing classes in `SubscriptionIT` @quarter-note

Reply:
Agree, these tests should be removed, but I would preserve the code from `PlainConsumerIT` and `PlainProducerIT` as a comment or some doc page which can be used as a straightforward example of how the SDK works internally for whose who are interested in implementing SDKs in other languages

Reply:
@quarter-note what do you think?

Reply:
Decided to keep `PlainProducerIT`/`PlainConsumerIT` it for now as an example of low-level interaction with Broker.
We might need to update it in the future when new protocol messages introduced.
This `testPayloadPlain` is ok to remove